R/preprocessing_io.R

```diff
@@ -1014,7 +1015,8 @@ liftover_bedpe = function(bedpe_file,
                           target_build = "grch37",
                           col_names,
                           col_types,
-                          standard_bed = FALSE){
+                          standard_bed = FALSE,
+                          print_head = TRUE){
```

Comment:
This is only for debugging, right? If so please set to FALSE by default. Also consider renaming the parameter to verbose to be consistent with other functions with a verbose print-happy debugging-mode

Reply:
So this function always prints the header of the lifted bedpe, this really clobbered the terminal when running `get_manta_by_samples` on a large cohort of samples (i.e all the missing samples). That is why I turned the `print(head)` statement into a parameter that is TRUE per default for the liftover function (i.e this function still works the same way as previously but allows us to silence this printing when we're performing liftover on multiple samples). I have changed the parameter name to be "verbose", but I think it should be TRUE per default for this function, to honour the way this function previously worked. Do you think this makes sense?

Reply:
No. verbosity should never occur by default. The old print(head) line was probably something that was there for debugging.

Reply:
Ok, I will address this then